docs/data-cookbooks/sips-workflow.md

```diff
@@ -0,0 +1,121 @@
+    "timestamp": 1533313798525,
+    "stats": {
+        "running": 0,
+        "completed": 0,
+        "failed": 2,
+        "total": 2
+    }
+}
+```
+
+2. We need to have a [provider](./setup.md#providers) from whom data can be ingested. Our provider is an S3 provider hosted in the `cumulus-test-internal` bucket.
+
+![](../images/sips-provider.png)
+
+3. Finally, let's create a [rule](./setup.md#rules). In this example we're just going to create a `onetime` throw-away rule that will be easy to test with. This rule will kick off the `DiscoverAndParsePdrs` workflow, which is the beginning of a Cumulus SIPS workflow.
+
+```
+{
+    "name": "s3_provider_rule",
+    "workflow": "DiscoverAndQueuePdrs",
+    "provider": "s3_provider",
+    "collection": {
+        "name": "MOD09GQ",
+        "version": "006"
+    },
+    "rule": {
+        "type": "onetime"
+    },
+    "state": "ENABLED",
+    "tags": [
+        "test"
+    ]
+}
+```
+
+**Note:** A list of configured workflows exists under the "Workflows" in the navigation bar on the Cumulus dashboard. Additionally, one can find a list of executions and their respective status in the "Executions" tab in the navigation bar.
+
+
+## DiscoverAndQueuePdrs Workflow
+
+This workflow will (as the name might suggest) discover pdrs and queue them to be processed. Duplicate pdrs will be dealt with according to the configured duplicate handling setting.
+
+1. DiscoverPdrs - [npm package](https://www.npmjs.com/package/@cumulus/discover-pdrs), [source](https://github.com/nasa/cumulus/tree/master/tasks/discover-pdrs)
+2. QueuePdrs - [npm package](https://www.npmjs.com/package/@cumulus/queue-pdrs), [source](https://github.com/nasa/cumulus/tree/master/tasks/queue-pdrs)
+
+![](../images/sips-discover-and-queue-pdrs-execution.png)
+
+
+## ParsePdr Workflow
+
+The ParsePdr workflow will parse a pdr, queue the specified granules (duplicates are handled according to the duplicate handling setting) and periodically check the status of those queued granules. This workflow will not succeed until all the granules included in the pdr are successfully ingested. If one of those fails, the ParsePdr worklfow will fail. **NOTE** that ParsePdr may spin up multiple IngestGranule workflows in parallel, depending on the granules included in the pdr.
+
+1. ParsePdr - [npm package](https://www.npmjs.com/package/@cumulus/parse-pdr), [source](https://github.com/nasa/cumulus/tree/master/tasks/parse-pdr)
+2. QueueGranules - [npm package](https://www.npmjs.com/package/@cumulus/queue-granules), [source](https://github.com/nasa/cumulus/tree/master/tasks/queue-granules)
+3. CheckStatus - [npm package](https://www.npmjs.com/package/@cumulus/pdr-status-check), [source](https://github.com/nasa/cumulus/tree/master/tasks/pdr-status-check)
+
+![](../images/sips-parse-pdr.png)
+
+
+## IngestGranule Workflow
+
+The IngestGranule workflow ingests, processes, and deals with updating CMR.
```

Comment:
I want to specify that it's a single granule. Maybe 'The IngestGranule workflow processes and ingests a granule and posts the granule metadata to CMR'